src/components/moderation/ModerationDetailsDialog.tsx

```diff
@@ -141,23 +141,24 @@ function ModerationDetailsDialogInner({
       {modcause?.type === 'label' && (
         <View style={[a.pt_lg]}>
           <Divider />
-          <Text style={[t.atoms.text, a.text_md, a.leading_snug, a.mt_lg]}>
-            {modcause.source.type === 'user' ? (
+          {modcause.source.type === 'user' ? (
+            <Text style={[t.atoms.text, a.text_md, a.leading_snug, a.mt_lg]}>
               <Trans>This label was applied by the author.</Trans>
-            ) : (
-              <Trans>
-                This label was applied by{' '}
-                <InlineLinkText
-                  label={desc.source || _(msg`an unknown labeler`)}
```

Comment:
And could you please provide the full sentence when `an unknown labeler` is displayed so that it can be localized?

Reply:
Could you verify (either by replying here or in the linked PR) that this fixes the problem here? https://github.com/bluesky-social/social-app/pull/5616/files#diff-70b5ca5a14b0410abcadacda7727faa8219f009b561533160ee75331fdd545da

(@tkusano thank you!)